docs/en/setup/backend/backend-redis-monitoring.md

```diff
@@ -36,3 +36,36 @@ You can customize your own metrics/expression/dashboard panel.
 The metrics definition and expression rules are found in `/config/otel-rules/redis`.
 The Redis dashboard panel configurations are found in `/config/ui-initialized-templates/redis`.
 
+## Collect sampled slow commands
+SkyWalking leverages [fluentbit](https://fluentbit.io/) or other log agents for collecting slow commands from Redis.
+
+### Data flow
+1. Execute [commands](../../../../test/e2e-v2/cases/redis/redis-exporter/scripts/slowlog.sh) periodically to collect slow logs from Redis and save the result locally.
+2. Fluent-bit agent collects slow logs from local file.
+3. fluent-bit agent sends data to SkyWalking OAP Server using native meter APIs via HTTP.
+4. The SkyWalking OAP Server parses the expression with [LAL](../../concepts-and-designs/lal.md) to parse/extract and store the results.
+
+### Set up
+1. Set up [fluentbit](https://docs.fluentbit.io/manual/installation/docker).
+2. Config fluentbit from [here](../../../../test/e2e-v2/cases/redis/redis-exporter/fluent-bit.conf) for Redis.
+3. Config slow log from [here](../../../../test/e2e-v2/cases/redis/redis-exporter/redis.conf) for Redis.
+4. Periodically execute the [commands](../../../../test/e2e-v2/cases/redis/redis-exporter/scripts/slowlog.sh).
```

Comment:
This is needed in production too? i meant this is not only for demo/e2e case? If so we should ask the users to check and modify the `slowlog.sh` file carefully before executing it, in the script file you hardcode the Redis host, log file location, etc., which might be not expected by the users.